img_proof/scripts/cli_utils.py

```diff
@@ -73,6 +73,24 @@ def archive_history_item(item, destination, no_color):
         )
 
 
+def cli_process_cpu_options(cxt, param, value):
+    cpu_options = []
+    if value:
+        try:
+            for cpu_option in value.split(','):
```

Comment:
Nitpick: Could be simplified to:

```
dict(option.split('=', 1) for option in cpu_options.split(','))
```

Then the resulting dictionary matches the API which simplifies that set of lines:

```
cpu_options = self.custom_args.get('cpu_options')
if cpu_options:
    kwargs['CpuOptions'] = cpu_options
```

Reply:
Fixed! The code is much simpler in the way you suggested. Thanks!